extract/Dockerfile

```diff
@@ -53,15 +53,18 @@ ENV GEODATA_POSTGRES_HOST="${GEODATA_POSTGRES_HOST:-pgsql}" \
     EXTERNAL_URL="${EXTERNAL_URL:-http://localhost:8080}"
 
 RUN apt update && apt upgrade -y && \
-    apt install -y curl python3 python3-geopandas python3-psycopg2 && \
+    apt install -y curl python3 python3-geopandas python3-psycopg2 python3-gdal && \
     apt install -y software-properties-common && \
-    add-apt-repository ppa:ubuntugis/ppa && \
     apt update && apt install -y python3-gdal && \
```

Comment:
As told previously, this apt update and duplicated python3-gdal installation aren't needed